lib/endpoint/find-parameters.js

```diff
@@ -216,3 +217,11 @@ function addPaginationParameters (state) {
     result.params.push(...PAGINATION_VARIABLES)
   })
 }
+
+function makeRelativeLinksAbsolute (state) {
+  const results = state.results[0].params
```

Comment:
you are only iterating trough the parameters of the first result now. You have to iterate trough all results.